student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
- [ ] Kindly make sure to also add age, name and parent permission to the initialize constructor. Like:
`def initialize(age, classroom, name = "Unknown", parent_permission = true)`

Reply:
noted and applied